### ISSUE TITLE
Backgrounds of moon and earth are not circles

### ISSUE THREAD

**Issue by sparr** [colobot/colobot#1584]
When watching the cut scene before the first moon mission, the earth and moon appear as ellipses. I suspect this is related to my screen resolution. The scenes should be cropped or letterboxed, not stretched.

**Comment by immibis** [comment]
I confirm the bug with the planets in the current dev commit (9417b916da9bbdb866d393feb82161679b613ff5). They stretch themselves to match the aspect ratio of the window.

The background stretches vertically to match your window height, but not horizontally, but it's not a huge problem as long as the background only contains some random stars.

**Comment by sparr** [comment]
@immibis do you also see the sky rotation not behaving properly when you turn your view in game?

**Comment by immibis** [comment]
@sparr Yes. That is another thing that will might require a full 3D sky to fix

**Comment by immibis** [comment]
I had a go at 3D planet rendering but I don't like the way it looks, especially because planets go higher up on the screen as they go closer to the edge of your screen (this is mathematically expected if you think about a circle rotating over your head, but it looks weird) and because the size of the planets changes with your FOV in first-person camera mode (this one might be a bug on my part).

Maybe instead we just keep the 2D picture but rotate it if the camera is rolled
